docs/mega-linter-runner.md

```diff
@@ -97,9 +102,6 @@ mega-linter-runner
 mega-linter-runner -p myFolder --fix
 ```
 
-```shell
-mega-linter-runner -r beta -e 'ENABLE=MARKDOWN,YAML' -e 'SHOW_ELAPSED_TIME=true'
-```
 
```

Comment:
We could include the analogue to your new example down here as well.

Reply:
done